pkg/resources/monitoring/image.go

```diff
@@ -2,23 +2,16 @@ package monitoring
 
 import (
 	"github.com/rancher/opni/pkg/resources"
-	corev1 "k8s.io/api/core/v1"
 	"k8s.io/client-go/util/retry"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 func (r *Reconciler) updateImageStatus() (bool, error) {
 	lg := r.logger
-	var image string
-	var pullPolicy corev1.PullPolicy
-	if imgOverride := r.mc.Spec.Cortex.Image.GetImageWithDefault(""); imgOverride != "" {
```

Comment:
It might still make sense to allow overriding the image, we could always open an issue to reintroduce this later

Reply:
Yeah, the grafana config here is very much still outdated and needs to be addressed. That's a whole other thing because we have to figure out what to do with the upstream grafana operator. I tried to make as few changes to it as I could in this pr.